README.md

```diff
@@ -1,75 +1,69 @@
+<img width="180" height="180" src="https://github.com/arl/gitmux/raw/readme-images/logo-white.png" align="right" />
+
 # Gitmux [![Build Status](https://travis-ci.com/arl/gitmux.svg?branch=master)](https://travis-ci.com/arl/gitmux) [![Go Report Card](https://goreportcard.com/badge/github.com/arl/gitmux)](https://goreportcard.com/report/github.com/arl/gitmux)
```

Comment:
Gitmux is cut in half on small screen mobile.
This could probably be solved by reducing the size of the logo